modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ModalAddObjectField.tsx

```diff
@@ -119,20 +119,31 @@ export function ModalAddObjectField({
 
 			const url = createResourceURL(baseResourceURL, {
 				objectDefinitionId: objectDefinitionResponse.id,
-				p_p_resource_id: '/object_definitions/get_object_field_types',
+				p_p_resource_id:
+					'/object_definitions/get_object_field_business_types',
 			}).href;
 
-			const objectFieldTypesResponse = await fetch(url, {
+			const objectFieldBusinessTypesResponse = await fetch(url, {
 				method: 'GET',
 			});
 
-			const {
-				objectFieldTypes,
-			} = (await objectFieldTypesResponse.json()) as {
-				objectFieldTypes: ObjectFieldType[];
+			let {
+				objectFieldBusinessTypes,
+			} = (await objectFieldBusinessTypesResponse.json()) as {
+				objectFieldBusinessTypes: ObjectFieldBusinessType[];
 			};
 
-			setObjectFieldTypes(objectFieldTypes);
+			objectFieldBusinessTypes = objectFieldBusinessTypes.filter(
+				(objectFieldBusinessType) => {
+					if (
+						objectFieldBusinessType.businessType !== 'Relationship'
+					) {
+						return objectFieldBusinessType;
+					}
+				}
+			);
+
+			setObjectFieldBusinessTypes(objectFieldBusinessTypes);
```

Comment:
Hey @igor-franca, you can place this filter directly inside the `setObjectFieldBusinessTypes` function